skins/cat17/src/sass/common/_variables.scss

```diff
@@ -19,6 +19,7 @@ $brand-primary-membership-dark: #4E9242;
 $brand-footer: #cfdbe2;
 $gray-light: #ddd;
 $brand-error: #812923;
+$brand-error-light: #e6d4d3;
 $brand-error-rgba: rgba(129, 41, 35, 0.2);
```

Comment:
I think we should take the opportunity and rename `$brand-error-rgba` to something more meaningful like `$brand-error-dark`.